package.json

```diff
@@ -13,11 +13,13 @@
   "scripts": {
     "export:docs": "node -r esbuild-register export/docs.ts",
     "export:overrides": "node -r esbuild-register export/overrides.ts",
+    "markdownlint": "npx markdownlint-cli '**/*.md' -i node_modules -i LICENSE.md",
```

Comment:
Weren't we going to replace this with prettier?

Reply:
I just pushed the recent changes for that. It includes the configurations to impact the existing Markdown as minimally as possible.